### PR TITLE
Add tests for worker_utils.py

### PR DESCRIPTION
**Pull request by matsujika**
[**The corresponding GCI task**](https://codein.withgoogle.com/dashboard/task-instances/5677433496272896/)

**Changes**:
- Add test cases for `worker_utils.py`
- Add a url for `update_submission_status` to `URLS` in `worker_utils.py`

**CC**: @vkartik97 utsav @Ram81 @RishabhJain2018

### CODE REVIEW

File: tests/unit/rl_worker/test_worker_util.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+import mock
+import requests
+import responses
+
+from unittest import TestCase
+
```
Comment (Ram81) [inline tests/unit/rl_worker/test_worker_util.py]: remove newline

File: tests/unit/rl_worker/test_worker_util.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+from scripts.workers.worker_util import EvalAI_Interface, URLS
```
Comment (Ram81) [inline tests/unit/rl_worker/test_worker_util.py]: reorder imports based on package name. Import on line 5 should come after line 7

Comment (takitsuse) [reply tests/unit/rl_worker/test_worker_util.py]: Okay, will fix that right away.

Comment (takitsuse) [reply tests/unit/rl_worker/test_worker_util.py]: @Ram81 Fixed :+1:

File: scripts/workers/worker_util.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "get_challenge_by_queue_name": "/api/challenges/challenge/queues/{}/",
     "get_challenge_phase_by_pk": "/api/challenges/challenge/{}/challenge_phase/{}",
     "update_submission_data": "/api/jobs/challenge/{}/update_submission/",
+    "update_submission_status": "/api/jobs/challenge/{}/update_submission/",
```
Comment (vkartik97) [inline scripts/workers/worker_util.py]: Can you remove this? I believe that this should be in a different PR

File: scripts/workers/worker_util.py [hunk 4]
```diff
@@ -92,7 +93,7 @@ def update_submission_data(self, data, challenge_pk, submission_pk):
         return response
 
     def update_submission_status(self, data, challenge_pk):
-        url = "/api/jobs/challenge/{}/update_submission/".format(challenge_pk)
+        url = URLS.get("update_submission_status").format(challenge_pk)
```
Comment (vkartik97) [inline scripts/workers/worker_util.py]: Same as https://github.com/Cloud-CV/EvalAI/pull/2612/files#r365990924

Comment (takitsuse) [reply scripts/workers/worker_util.py]: Okay, hardcode URL for now.

Comment (takitsuse) [reply scripts/workers/worker_util.py]: I'm reluctant to hardcode things though.

Comment (takitsuse) [reply scripts/workers/worker_util.py]: @vkartik97 Removed :+1:

Comment (vkartik97) [reply scripts/workers/worker_util.py]: > I'm reluctant to hardcode things though.

How about you open an issue for this as you discovered it?

Comment (takitsuse) [reply scripts/workers/worker_util.py]: That's a good idea. But I'll create it later since it is 5 AM here (I was woken up by earthquake).

**Comment by takitsuse** [comment]
@vkartik97 I created the PR at https://github.com/Cloud-CV/EvalAI/pull/2617. Please take a look.
